speechbrain/lobes/models/wav2vec.py

```diff
@@ -0,0 +1,315 @@
+import math
```

Comment:
Function like negative sampling and collate function can go there, if you think that they are only linked to w2v2

Reply:
okay will move the collate function